src/sentry/replaystore/django/backend.py

```diff
@@ -1,21 +1,33 @@
 from __future__ import annotations
 
+from datetime import datetime
 from typing import List, Tuple
 
-from sentry.replaystore.base import ReplayNotFound, ReplayStore
+from sentry.replaystore.base import ReplayDataType, ReplayNotFound, ReplayStore
 from sentry.replaystore.django.models import Replay
 from sentry.utils.strings import compress
 
 
 class DjangoReplayStore(ReplayStore):
-    def _get_rows(self, key: str) -> List[Tuple[str, bytes]]:
-        data = Replay.objects.filter(id__startswith=key)
+    def _get_rows(self, replay_id: str) -> List[Tuple[ReplayDataType, bytes]]:
+        data = Replay.objects.filter(replay_id=replay_id)
         if len(data) == 0:
             raise ReplayNotFound
-        return [(row.id, row.data) for row in data]
+        return [(row.replay_data_type, row.data) for row in data]
 
-    def _set_bytes(self, key: str, value: bytes) -> None:
-        replay = Replay.objects.create(id=key, data=compress(value))
+    def _set_bytes(
+        self,
+        replay_id: str,
+        replay_data_type: ReplayDataType,
+        timestamp: datetime,
+        data: bytes,
+    ) -> None:
+        replay = Replay.objects.create(
+            replay_id=replay_id,
+            replay_data_type=replay_data_type,
+            timestamp=timestamp,
+            data=compress(data),
+        )
```

Comment:
Based on the bigtable implementation I assume we want this to be unique. If that's the case, should we use `get_or_create` here to avoid making duplicate rows?